source/includes/compatibility-table.rst

```diff
@@ -0,0 +1,11 @@
+.. list-table::
+   :header-rows: 1
+   :widths: 50 25 25
+
+   * - {+provider-long+}
+     - Entity Framework Core
```

Comment:
S: We already have a constant for Entity Framework (framework). Can we add one for Entity Framework Core that equals "{+framework+} Core"? Can then use throughout this PR.